src/studio/nuxeo-studio/working-in-studio/branding.md

```diff
@@ -391,168 +392,93 @@ history:
-
-1.  Select a background video or a background image. Be careful not to use an image too heavy to a quick display for the users.
-2.  Define the options for the video or the image, for example to mute the video or the position (in pixel or position) of the image, if you know that it will be large enough for the huge screen resolutions.
-3.  Or make the background cover all the screen (with CSS3 / recent browsers).
-4.  Pick a color that will be displayed if the background image is too small and not covering all the screen.
-
-That's it!
-
-</td></tr></tbody></table></div>
-
-Most of our customers apply the following strategies for their login page background image:
-
-*   A picture covering all the screen. This is the strategy chosen by Nuxeo.
-*   Or a logo with transparency as the background image, aligned on the top left (Horizontal alignment = top, Vertical alignment = left) + a background color.
-
-With a web designer's help, you can have many other possibilities.
-
-### Additional Options
-
-<div class="table-scroll"><table class="hover"><tbody><tr><th colspan="1">Login page layout</th><th colspan="1">Studio Configuration</th><th colspan="1">Role</th></tr><tr><td colspan="1">![]({{file name='login_layout.png'}} ?w=300,border=true,thumbnail=true)</td><td colspan="1">
-
-![]({{file name='login_remove_boxes.png'}} ?w=310,border=true)
-
-</td><td colspan="1">To let you customize the login screen completely, the news and footer can be removed.</td></tr><tr><td colspan="1">
-
-![]({{file name='login_box.png'}} ?w=310,border=true)
-
-</td><td colspan="1">
-
-If you have a simple color used as the background image in the login page, you can add a color to the login box and to the button.
-
-This option must be used carefully: don't use heavy patterns or things that will make the readability difficult or hamper the interpretation. The login box style must stay very simple and light to keep the login action as a user-friendly step.
-
-</td></tr></tbody></table></div>
-
-{{! /multiexcerpt}}
-
-{{> anchor 'css'}}
-
-## CSS Editor Tab
-
-The CSS Editor is dedicated to web designers who know CSS and its syntax. It's the place to declare your new CSS rules or override existing rules that will be used into the Nuxeo main tabs: Home, Document Management and Admin Center.
-
-{{#> callout type='info' }}
-
-Have a look at the [Nuxeo UI Style Guide](http://showcase.nuxeo.com/nuxeo/styleGuide/) to see all the existing CSS rules used by default in your platform, and understand more about readability, usability, layout design, and user experience.
+-   A picture covering all the screen. This is the strategy chosen by Nuxeo.
+-   A logo with a transparent background with a background color.
```

Comment:
Maybe `background and` instead of `background with`?

Reply:
Done